sdk/identity/Azure.Identity.Broker/src/Azure.Identity.Broker.csproj

```diff
@@ -13,7 +13,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" VersionOverride="1.14.0-beta.4" />
+    <PackageReference Include="Azure.Identity" VersionOverride="1.15.0-beta.1" />
```

Comment:
Do we still need this version override?

Reply:
Yes, the MacOS constant was shipped in the beta release, and the central package version points at the latest GA